neural_speed/__init__.py

```diff
@@ -94,6 +94,10 @@ def init(self, model_name, not_quant=False, use_cache=False,
                 quant_desc += "_pc"
             else:
                 quant_desc += "_g{}".format(group_size)
+        if use_gptq:
+            quant_desc = "gptq"
+        if use_awq:
+            quant_desc = "awq"
```

Comment:
why not?

```suggestion
        if use_gptq:
            quant_desc += "_gptq"
        if use_awq:
            quant_desc += "_awq"
```

Reply:
quant_desc includes weight_dtype int 4/int 8, but for awq/gptq, we don't know the weight dtype from init

Reply:
> quant_desc includes weight_dtype int 4/int 8, but for awq/gptq, we don't know the weight dtype from init

```awq.bin``` seems not meaningful enough, any other info to differ the model file?

Reply:
There is currently no convenient method, the file here is before convert, but info is available after convert